atmel-samd/boards/metro_m0_express/pins.c

```diff
@@ -26,9 +26,9 @@ STATIC const mp_map_elem_t board_global_dict_table[] = {
   { MP_OBJ_NEW_QSTR(MP_QSTR_SDA), (mp_obj_t)&pin_PA22 },
   { MP_OBJ_NEW_QSTR(MP_QSTR_SCL), (mp_obj_t)&pin_PA23 },
   { MP_OBJ_NEW_QSTR(MP_QSTR_NEOPIXEL), (mp_obj_t)&pin_PA30 },
-  { MP_OBJ_NEW_QSTR(MP_QSTR_FLASH_SCK),  (mp_obj_t)&pin_PB11 },
-  { MP_OBJ_NEW_QSTR(MP_QSTR_FLASH_MOSI), (mp_obj_t)&pin_PB10 },
-  { MP_OBJ_NEW_QSTR(MP_QSTR_FLASH_MISO), (mp_obj_t)&pin_PA12 },
-  { MP_OBJ_NEW_QSTR(MP_QSTR_FLASH_CS),   (mp_obj_t)&pin_PA13 },
+  { MP_OBJ_NEW_QSTR(MP_QSTR_EXT_SCK),  (mp_obj_t)&pin_PB11 },
+  { MP_OBJ_NEW_QSTR(MP_QSTR_EXT_MOSI), (mp_obj_t)&pin_PB10 },
+  { MP_OBJ_NEW_QSTR(MP_QSTR_EXT_MISO), (mp_obj_t)&pin_PA12 },
+  { MP_OBJ_NEW_QSTR(MP_QSTR_EXT_CS),   (mp_obj_t)&pin_PA13 },
```

Comment:
Please remove EXT_ from the names and remove the CS line. Its still the CS line for the flash and not on the 2x3 SPI connector.